models/reviews.models.js

```diff
@@ -57,7 +58,32 @@ exports.updateVotes = (review_id, votes) => {
     });
 };
 
-exports.getAllReviews = (category) => {
+exports.getAllReviews = ({ sort_by, category, order }) => {
+  const validSorts = [
+    "owner",
+    "title",
+    "review_id",
+    "category",
+    "review_img_url",
+    "created_at",
+    "votes",
+    "designer",
+    "comment_count",
+  ];
+  if (!sort_by) sort_by = "created_at";
```

Comment:
You can also handle this by assigning the sort_by in the parameters bit - like you did on OOP